inference-engine/src/mkldnn_plugin/mkldnn_node.h

```diff
@@ -361,6 +361,8 @@ class MKLDNNNode {
         return engine;
     }
 
+    bool isExecutable();
+
```

Comment:
What do you think, may be it is better to make this method virtual with `return true` by default and override it in nodes where it makes sense. It will help us avoid introduction such a property as `ExecutableType executable` and other ongoing complexity. We also do not need caching since `isExecutable()` is called once in `ExtractConstantNodes`. BTW probably we should rename `ExtractConstantNodes` method since its responsibly was extended.